Client/src/app/components/authentication/login/login.component.css

```diff
@@ -0,0 +1,34 @@
+.otherServices {
+  position: absolute;
+  left: 40%;
+  top: 50px;
+  height: min-content;
+  width: min-content;
+  padding: 20px;
+}
+
```

Comment:
We have bootstrap installed and also font-awesome. So unless bootstrap does not have what you need, theres no need to define your own styling. How will this look on another device? Currently the values are very hard coded.